source/core/types/ut_output_stream.tpb

```diff
@@ -23,3 +23,8 @@ create or replace type body ut_output_stream as
 
 end;
 /
+--we need to use plsql optimize level=1 to prevent
```

Comment:
How about changing the " for loop" to open-loop-fetch-close? I thought optimization leaves it as is. This way altering plsql_optimizate_level is not needed (but should be rechecked)